File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Bass/BAS0110U00GrdJohapListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Bass
{
    [Serializable]
    public class BAS0110U00GrdJohapListItemInfo
    {
        private String _johap;
        private String _startDate;
        private String _johapName;
        private String _johapGubun;
        private String _zipCode1;
        private String _zipCode2;
        private String _address;
        private String _cd;
        private String _boheomjaNo;
        private String _dodobuhyeunNo;
        private String _lawNo;
        private String _address1;
        private String _tel;
        private String _giho;
        private String _remark;
        private String _checkDigitYn;
        private String _rowState;

        public String Johap
        {
            get { return this._johap; }
            set { this._johap = value; }
        }

        public String StartDate
        {
            get { return this._startDate; }
            set { this._startDate = value; }
        }

        public String JohapName
        {
            get { return this._johapName; }
            set { this._johapName = value; }
        }

        public String JohapGubun
        {
            get { return this._johapGubun; }
            set { this._johapGubun = value; }
        }

        public String ZipCode1
        {
            get { return this._zipCode1; }
            set { this._zipCode1 = value; }
        }

        public String ZipCode2
        {
            get { return this._zipCode2; }
            set { this._zipCode2 = value; }
        }

        public String Address
        {
            get { return this._address; }
            set { this._address = value; }
        }

        public String Cd
        {
            get { return this._cd; }
            set { this._cd = value; }
        }

        public String BoheomjaNo
        {
            get { return this._boheomjaNo; }
            set { this._boheomjaNo = value; }
        }

        public String DodobuhyeunNo
        {
            get { return this._dodobuhyeunNo; }
            set { this._dodobuhyeunNo = value; }
        }

        public String LawNo
        {
            get { return this._lawNo; }
            set { this._lawNo = value; }
        }

        public String Address1
        {
            get { return this._address1; }
            set { this._address1 = value; }
        }

        public String Tel
        {
            get { return this._tel; }
            set { this._tel = value; }
        }

        public String Giho
        {
            get { return this._giho; }
            set { this._giho = value; }
        }

        public String Remark
        {
            get { return this._remark; }
            set { this._remark = value; }
        }

        public String CheckDigitYn
        {
            get { return this._checkDigitYn; }
            set { this._checkDigitYn = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public BAS0110U00GrdJohapListItemInfo() { }

        public BAS0110U00GrdJohapListItemInfo(String johap, String startDate, String johapName, String johapGubun, String zipCode1, String zipCode2, String address, String cd, String boheomjaNo, String dodobuhyeunNo, String lawNo, String address1, String tel, String giho, String remark, String checkDigitYn, String rowState)
        {
            this._johap = johap;
            this._startDate = startDate;
            this._johapName = johapName;
            this._johapGubun = johapGubun;
            this._zipCode1 = zipCode1;
            this._zipCode2 = zipCode2;
            this._address = address;
            this._cd = cd;
            this._boheomjaNo = boheomjaNo;
            this._dodobuhyeunNo = dodobuhyeunNo;
            this._lawNo = lawNo;
            this._address1 = address1;
            this._tel = tel;
            this._giho = giho;
            this._remark = remark;
            this._checkDigitYn = checkDigitYn;
            this._rowState = rowState;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/bass/BAS0110U00GrdJohapListItemInfo.java
package nta.med.data.model.ihis.bass;

import java.util.Date;

public class BAS0110U00GrdJohapListItemInfo {
	private String johap;
	private Date startDate;
	private String johapName;
	private String johapGubun;
	private String zipCode1;
	private String zipCode2;
	private String address;
	private String cd;
	private String boheomjaNo;
	private String dodobuhyeunNo;
	private String lawNo;
	private String address1;
	private String tel;
	private String giho;
	private String remark;
	private String checkDigitYn;
	private String rowState;

	public BAS0110U00GrdJohapListItemInfo(String johap, Date startDate,
			String johapName, String johapGubun, String zipCode1,
			String zipCode2, String address, String cd, String boheomjaNo,
			String dodobuhyeunNo, String lawNo, String address1, String tel,
			String giho, String remark, String checkDigitYn, String rowState) {
		super();
		this.johap = johap;
		this.startDate = startDate;
		this.johapName = johapName;
		this.johapGubun = johapGubun;
		this.zipCode1 = zipCode1;
		this.zipCode2 = zipCode2;
		this.address = address;
		this.cd = cd;
		this.boheomjaNo = boheomjaNo;
		this.dodobuhyeunNo = dodobuhyeunNo;
		this.lawNo = lawNo;
		this.address1 = address1;
		this.tel = tel;
		this.giho = giho;
		this.remark = remark;
		this.checkDigitYn = checkDigitYn;
		this.rowState = rowState;
	}

	public String getJohap() {
		return johap;
	}

	public void setJohap(String johap) {
		this.johap = johap;
	}

	public Date getStartDate() {
		return startDate;
	}

	public void setStartDate(Date startDate) {
		this.startDate = startDate;
	}

	public String getJohapName() {
		return johapName;
	}

	public void setJohapName(String johapName) {
		this.johapName = johapName;
	}

	public String getJohapGubun() {
		return johapGubun;
	}

	public void setJohapGubun(String johapGubun) {
		this.johapGubun = johapGubun;
	}

	public String getZipCode1() {
		return zipCode1;
	}

	public void setZipCode1(String zipCode1) {
		this.zipCode1 = zipCode1;
	}

	public String getZipCode2() {
		return zipCode2;
	}

	public void setZipCode2(String zipCode2) {
		this.zipCode2 = zipCode2;
	}

	public String getAddress() {
		return address;
	}

	public void setAddress(String address) {
		this.address = address;
	}

	public String getCd() {
		return cd;
	}

	public void setCd(String cd) {
		this.cd = cd;
	}

	public String getBoheomjaNo() {
		return boheomjaNo;
	}

	public void setBoheomjaNo(String boheomjaNo) {
		this.boheomjaNo = boheomjaNo;
	}

	public String getDodobuhyeunNo() {
		return dodobuhyeunNo;
	}

	public void setDodobuhyeunNo(String dodobuhyeunNo) {
		this.dodobuhyeunNo = dodobuhyeunNo;
	}

	public String getLawNo() {
		return lawNo;
	}

	public void setLawNo(String lawNo) {
		this.lawNo = lawNo;
	}

	public String getAddress1() {
		return address1;
	}

	public void setAddress1(String address1) {
		this.address1 = address1;
	}

	public String getTel() {
		return tel;
	}

	public void setTel(String tel) {
		this.tel = tel;
	}

	public String getGiho() {
		return giho;
	}

	public void setGiho(String giho) {
		this.giho = giho;
	}

	public String getRemark() {
		return remark;
	}

	public void setRemark(String remark) {
		this.remark = remark;
	}

	public String getCheckDigitYn() {
		return checkDigitYn;
	}

	public void setCheckDigitYn(String checkDigitYn) {
		this.checkDigitYn = checkDigitYn;
	}

	public String getRowState() {
		return rowState;
	}

	public void setRowState(String rowState) {
		this.rowState = rowState;
	}

}
